scripts/build/companion_libs/330-gettext.sh

```diff
@@ -118,6 +118,14 @@ do_gettext_backend() {
         --disable-csharp                                    \
         --without-emacs                                     \
         --disable-openmp                                    \
+        --with-included-libxml                              \
+        --with-included-gettext                             \
+        --with-included-glib                                \
+        --with-included-libcroco                            \
+        --with-included-libunistring                        \
+        --with-libncurses-prefix="${prefix}"                \
```

Comment:
Here's a link to an example of the failure:
https://travis-ci.org/crosstool-ng/crosstool-ng/jobs/184700382

I wasn't able to reproduce the *failure* since my host has recent enough libxml2. But I did reproduce gettext linking to the host libxml2.